addon/mixins/time-series-labeler.js

```diff
@@ -159,6 +160,20 @@ export default Ember.Mixin.create({
     }
   }),
 
+  // A function that can be passed in if there's tick labels we specifically
+  // wish to filter out (for example first, last, overflow, etc)
+  // NOTE: This function filters the ticks after centering them (when specified),
+  // meaning that the functionality here is not trivially replicable simply by
+  // modifying `this.filterLabels` in the `tickLabelerFn` implementation.
```

Comment:
I'd still use jsdoc syntax even though most of ember-charts doesn't, coffeescript to javascript migration left over some funky stuff.